### PR TITLE
Small updates to reagent containers

### PR DESCRIPTION
**Pull request by BeagleGaming1**
# About the pull request
Removes APTFT verb, leaves only the action
Updates reagent container code

# Explain why it's good for the game
Better code

# Changelog

:cl:
del: Removes APTFT verb from reagent containers
code: Updated reagent container switch transfer amount and examination code
/:cl:

### CODE REVIEW

File: code/game/objects/items/reagent_containers/hypospray.dm [hunk 1]
```diff
@@ -25,6 +25,10 @@
 	var/next_inject = 0
 	var/inject_cd = 0.75 SECONDS
 
+/obj/item/reagent_container/hypospray/Destroy()
+	. = ..()
+	QDEL_NULL(mag)
```
Comment (Zonespace27) [inline code/game/objects/items/reagent_containers/hypospray.dm:30]: call parent after code in Destroy

File: code/game/objects/items/reagent_containers/reagent_container.dm [hunk 2]
```diff
@@ -1,114 +1,108 @@
...
+	if(has_set_transfer_action && LAZYLEN(possible_transfer_amounts))
+		LAZYADD(actions_types, /datum/action/item_action/reagent_container/set_transfer_amount)
 	. = ..()
-	if(!possible_transfer_amounts)
-		verbs -= /obj/item/reagent_container/verb/set_APTFT //which objects actually uses it?
 	create_reagents(volume)
 
+/obj/item/reagent_container/Destroy()
+	possible_transfer_amounts = null
+	return ..()
```
Comment (Zonespace27) [inline code/game/objects/items/reagent_containers/reagent_container.dm:31]: not needed

File: code/game/objects/items/reagent_containers/reagent_container.dm [hunk 3]
```diff
@@ -1,114 +1,108 @@
...
+		if(total_reagent_desc != "")
+			total_reagent_desc += ", "
+		total_reagent_desc += "[current_reagent.name] ([current_reagent.volume]u)"
 
+	return total_reagent_desc
 
 /datum/action/item_action/reagent_container/set_transfer_amount
+	name = "Set Transfer Amount"
+	/// The container that transfer amount will be set on
+	var/obj/item/reagent_container/container
```
Comment (Zonespace27) [inline code/game/objects/items/reagent_containers/reagent_container.dm:92]: null on destroy

File: code/game/objects/items/reagent_containers/reagent_container.dm [hunk 4]
```diff
@@ -1,114 +1,108 @@
...
-/obj/item/reagent_container/Destroy()
-	possible_transfer_amounts = null
-	return ..()
+	if(user.can_see_reagents())
+		reagent_desc += get_reagent_list_text()
+		return SPAN_INFO(reagent_desc)
+
+	if(!transparent)
+		return
+	if(user != loc && get_dist(user, src) >2)
```
Comment (Zonespace27) [inline code/game/objects/items/reagent_containers/reagent_container.dm:55]:
```suggestion
	if(user != loc && !in_range(src, user)
```

**Review by Zonespace27 — DISMISSED** [review]

**Review by harryob — APPROVED** [review]

**Pull request merged**